app/assets/javascripts/components/locales/en.jsx

```diff
@@ -94,7 +94,6 @@ const en = {
   "navigation_bar.public_timeline": "Federated timeline",
   "notification.favourite": "{name} favourited your status",
   "notification.follow": "{name} followed you",
-  "notification.mention": "{name} mentioned you",
```

Comment:
Other `notification.*` strings are used in `app/assets/javascripts/components/features/notifications/components/notification.jsx`, but `notification.mention` is not used. Also I can't found use case in `app/assets/javascripts/**.jsx`.

However, I also cannot found use case in the tree at the time this line introduced (#1867), so it may be used in other places. How do you think @Gargron?